tt_eager/tt_dnn/op_library/backward/backward_ops.hpp

```diff
@@ -236,6 +236,8 @@ std::vector<Tensor> log2_bw(const Tensor& grad, const Tensor& input, const Memor
 std::vector<Tensor> ge_bw(const Tensor& grad, const MemoryConfig& output_mem_config = operation::DEFAULT_OUTPUT_MEMORY_CONFIG);
 
 std::vector<Tensor> le_bw(const Tensor& grad, const MemoryConfig& output_mem_config = operation::DEFAULT_OUTPUT_MEMORY_CONFIG);
+
+std::vector<Tensor> mvlgamma_bw(const Tensor& grad, const Tensor& input, const MemoryConfig& output_mem_config = operation::DEFAULT_OUTPUT_MEMORY_CONFIG);
```

Comment:
Where is the mvlgamma for the forward pass?

Reply:
Hi @eyonland, in the forward implementation, it is named as 'multigammaln'. - Tensor multigammaln(const Tensor& a, const MemoryConfig& output_mem_config = operation::DEFAULT_OUTPUT_MEMORY_CONFIG);

Reply:
Ref : https://pytorch.org/docs/stable/generated/torch.mvlgamma.html

Reply:
Hey @bharane-ab !   Why is there a name change here?

Reply:
I have updated the doc and renamed the op as multigammaln_bw